<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_adiciona_edicao</class>
 <widget class="QMainWindow" name="janela_adiciona_edicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QLineEdit{
    background: #ffffff;
    border: 2px solid #4A90E2;
    border-radius: 8px;
    padding: 5px 10px;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 16px;
    color: #333333;
}

QLineEdit:focus{
    border-color: #53A3FF;
    background: #E6F7FF; 
}

QLineEdit:disabled {
    background: #f0f0f0;
    border: 2px solid #cccccc;
    color: #999999;
}

QRadioButton {
    background: #ffffff;
    border: 2px solid #4A90E2;
    border-radius: 8px;
    padding: 5px 10px;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 14px;
    color: #000000;
}

QRadioButton:checked {
    background: #4A90E2; 	/* Fundo azul quando selecionado */
    border-color: #53A3FF; 	/* Borda com azul mais claro */
    color: #FFFFFF;					 /* Texto branco */
}

QRadioButton:hover {
    border-color: #53A3FF;	 /* Borda azul claro ao passar o mouse */
    background: #E6F7FF;		 /* Fundo azul claro */
    color: #1D08FF;					 /* Texto branco */
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 14px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QPushButton:hover{
    background: #53A3FF;
    border: 2px solid #fff;
    border-radius: 12px;
    padding: 10px 20px;
}

QDateEdit {
    background: #ffffff;
    border: 2px solid #4A90E2;
    border-radius: 8px;
    padding: 5px 10px;

    font-family: &quot;Segoe UI&quot;;
    font-weight: bold;
    font-size: 16px;
    color: #333333;
}

QDateEdit:focus {
    border-color: #53A3FF;
    background: #E6F7FF;
}

QDateEdit::drop-down {
    border: none;
    width: 20px;
    background: #4A90E2;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QDateEdit::drop-down:hover {
    background: #53A3FF;
}

QDateEdit:disabled {
    background: #f0f0f0;
    border: 2px solid #cccccc;
    color: #999999;
}

QDateEdit::down-arrow {
    width: 10px;
    height: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="_txt_peso">
          <property name="text">
           <string>Digite seu peso: (kg)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_line_peso">
          <property name="cursor">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="_txt_altura">
          <property name="text">
           <string>Digite sua altura: (m)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_line_altura"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="_txt_datanascimento">
          <property name="text">
           <string>Data de Nascimento:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="_date_nascimento">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="_txt_objetivo">
      <property name="text">
       <string>Marque seu objetivo principal:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="_rad_hipertrofia">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Hipertrofia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_rad_emagrecimento">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Emagrecimento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_rad_mobilidade">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Mobilidade</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="_rad_ganhoforca">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Ganho de for√ßa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_rad_condicionamento">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Condicionamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_rad_disciplina">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Disciplina</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="_btn_confirma">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Confirmar Dados</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
